### PR TITLE
[Feature] Show Disk Space and Free Disk Space #89

### PR DESCRIPTION
**Pull request by beathorst**
You need the PR vor t3monitor_client as well
https://github.com/georgringer/t3monitoring/issues/89

### CODE REVIEW

File: Classes/ViewHelpers/Format/DiskSpaceViewHelper.php [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+class DiskSpaceViewHelper extends AbstractViewHelper
+{
+
+    /**
+     * @param int $bytes
+     * @return string
+     */
+    public function render($bytes = 0)
+    {
+        $bytes = $bytes ?: $this->renderChildren();
```
Comment (georgringer) [inline Classes/ViewHelpers/Format/DiskSpaceViewHelper.php]: why not using \TYPO3\CMS\Fluid\ViewHelpers\Format\BytesViewHelper ?

File: Configuration/TCA/tx_t3monitoring_domain_model_client.php [hunk 2]
```diff
@@ -113,6 +114,26 @@
...
             ],
         ],
+        'disk_total_space' => [
+            'exclude' => 1,
+            'label' => 'LLL:EXT:t3monitoring/Resources/Private/Language/locallang.xlf:tx_t3monitoring_domain_model_client.disk_total_space',
+            'config' => [
+                'readOnly' => true,
+                'type' => 'input',
+                'size' => 5,
+                'eval' => 'trim'
```
Comment (georgringer) [inline Configuration/TCA/tx_t3monitoring_domain_model_client.php]: eval "int" would maybe fit better?

File: ext_tables.sql [hunk 3]
```diff
@@ -12,6 +12,8 @@ CREATE TABLE tx_t3monitoring_domain_model_client (
 	email varchar(255) DEFAULT '' NOT NULL,
 	php_version varchar(255) DEFAULT '' NOT NULL,
 	mysql_version varchar(255) DEFAULT '' NOT NULL,
+	disk_free_space varchar(255) DEFAULT '' NOT NULL,
```
Comment (georgringer) [inline ext_tables.sql]: why not using bytes and integer?

**Comment by beathorst** [comment]
change the comments

**Comment by georgringer** [comment]
what about using integers instead of varchar?

**Comment by beathorst** [comment]
I changed from varchar to bigint.

**Pull request merged**